src/cityreader/cityreader.py

```diff
@@ -16,26 +27,38 @@
 # should not be loaded into a City object.
 cities = []
 
+
+# def zero_strip(n):
+#     s = str(Decimal(n))
+#     return float(s.rstrip('0').rstrip('.') if '.' in s else s)
+
+
 def cityreader(cities=[]):
-  # TODO Implement the functionality to read from the 'cities.csv' file
-  # For each city record, create a new City instance and add it to the 
-  # `cities` list
-    
+    # TODO Implement the functionality to read from the 'cities.csv' file
+    # For each city record, create a new City instance and add it to the
+    # `cities` list
+    with open('cities.csv', 'r') as csv_file:
```

Comment:
`with` auto closes the file after the code block is executed. `as` assigns the output of the open function.